src/utils/withWidth.js

```diff
@@ -22,8 +22,11 @@ export default function withWidth(options = {}) {
         width: SMALL,
       };
 
-      componentDidMount() {
-        this.updateWidth();
+      componentWillMount() {
```

Comment:
For my particular case of rendering the `<Snackbar />` not rendering server side is fine.

If the `withWidth` HoC is used in many components it will remove some of the server side render benefits but I'd prefer that tradeoff over the checksum mismatch.

Thanks Oliver.